lib/mm_network.pm

```diff
@@ -66,11 +65,9 @@ sub configure_dhcp {
     type_string "NIC=`grep \$MAC /sys/class/net/*/address |cut -d / -f 5`;";
     type_string("echo \"STARTMODE='auto'\nBOOTPROTO='dhcp'\n\" > /etc/sysconfig/network/ifcfg-\$NIC;");
     type_string("done\n");
```

Comment:
At least "done\n" should be done with `script_run` to avoid race conditions on a fast console (i.e. virtio console). In this case possibly even VNC. Although `save_screenshot` won't work on a text console anyway (not that it would be needed, so it could fail silently).